Make a note that bitsandbytes only support Cuda and Linux. (#284)
Lightning-AI/lit-gpt: tutorials/quantize.md

## `bnb.nf4`

> Enabled with [bitsandbytes](https://github.com/TimDettmers/bitsandbytes). Check out the [paper](https://arxiv.org/abs/2305.14314v1) to learn more about how it works.

Reviewer: Thanks for fixing the  typo. Can you fix it in the other places of this file? Or the repo if you find any others

Author: Sure. I'll do it tomorrow.
What about trailing whitespaces, should I remove them too?

Reviewer: Sure if you find any

Author: Done

> > **Note**: for now `bitsandbytes` only supports `CUDA` device and `Linux` operating system.

Reviewer: ```suggestion
> **Note**: `bitsandbytes` only supports `CUDA` devices and the `Linux` operating system.
```

nit: we don't have any visibility on the timeline. "for now" makes it sound like it might change

Author: A little bit of hope doesn't hurt :)
But in all seriousness I agree, it's better to be careful with words.

Though I have a feeling that as soon as PyTorch is shipped with CUDA 12 we will revisit it.
